pkg/apis/seedmanagement/v1alpha1/defaults_gardenlet_test.go

```diff
@@ -148,6 +163,11 @@ var _ = Describe("Defaults", func() {
 											Namespace: "bar",
 										},
 									},
+									Ingress: &gardencorev1beta1.Ingress{
+										Controller: gardencorev1beta1.IngressController{
+											Kind: "nginx",
+										},
+									},
```

Comment:
Lets verify that an already specified ingress configuration would not be overwritten instead